src/localplayer.h

```diff
@@ -144,7 +144,9 @@ class LocalPlayer : public Player
 	void setCollisionbox(const aabb3f &box) { m_collisionbox = box; }
 
 private:
+	// clang-format off
```

Comment:
i dont understand why clang-format doesn't like this 89 chars line, disable locally